src/app/lib/providers/generic.js

```diff
@@ -17,14 +17,14 @@
         var name = PO.prototype.config ? PO.prototype.config.name : null;
 
         if (!name) {
-            return win.error(PO, PO.prototype.config, 'doesnt have a name');
+            return console.error(PO, PO.prototype.config, 'doesnt have a name');
         }
 
         if (registry[name]) {
-            return win.error('double definition of', name, PO, PO.prototype.config, 'is the same as', registry[name]);
+            return console.error('double definition of', name, PO, PO.prototype.config, 'is the same as', registry[name]);
         }
 
-        win.debug('added', name, 'to provider registry');
+        //console.log('Added %s to provider registry', name);
```

Comment:
It's spam to me. If you disagree, uncomment it: what bothers me is that it's repeated every load, without any real need for it. I mean, it's in package.json, it should be loaded, I don't see why the log is needed (except when working on it). Logging isn't commenting